### DISCUSSION TITLE
Fix issue list in mobile

### DISCUSSION THREAD

**Discussion by Atomicwallets** [community/community#3607] [Mobile]
A have 57 issues in my private repo, but in mobile app I see only 4 random issues.
Android 9.

**Comment by aconsuegra** [comment]
Hey @Atomicwallets, thanks for reaching out!

Can you reproduce this bug in other repos? Specially in a public repo? Or is it only happening in a private one?

**Reply by Atomicwallets** [reply]
Hi, only in private repo. I don't have an open repo.
